### DISCUSSION TITLE
Make your linkedIn profile better | Reviewing Your LinkedIn profile live stream by Siddharth Dayalwal

### DISCUSSION THREAD

**Discussion by jai-dewani** [EddieHubCommunity/support#1645] [General]
### Link to the live stream -> https://www.youtube.com/watch?v=OrrTA-UiRaY

Linked's mission -> Connect the world's professionals to make them more productive and successful 
Linked's vision -> Create economic opportunity for every member of the global workforce 

### There are some of the tips that Siddharth shared, with some personal opinions sprinkled in
- Have a proper photo on your profile, keep it above the abdominal region, and made sure your face is well lit 
- Make sure you have added a recorded pronunciation of your name 
- Add your industry 
- Show that you are open to work (if you are) 
- Add a good about me. It's your elevator pitch, add your achievements and things you enjoy working on and how or why people should people contact you. 
- Add your featured content so people know what you are proud of, your portfolio, or a post you made. 
- Definatly add a description for all your experiences, it helps others understand what you did  (because fullstack developer is a very vast role and you can't do everything in a two month internship, so showcase what you did do) so they can reach out to you more confidently
- Add volunteer experience, to showcase that you do things outside your job or work as well.
- Add at least your top 5 skills and 10 is a good number to start with. 

- Have recommendations, ask your managers, senior team members to recommend you after you have worked with them. 

### Points that came up while reviewing other profiles 
- If you are a community member, add that to your volunteer experience and not in work experience. 
- Have more nouns on your bio than verbs 
- Don't make your about section too big, have a max of 2-3 paragraphs and around 100 works should be enough
- Add a specific skill instead of just `programming`

**Comment by eddiejaoude** [comment]
Thanks @jai-dewani for documenting 💥
